Getting Started.md

```diff
@@ -0,0 +1,73 @@
+
```

Comment:
While writing a project's description, always begin with the project's name. Leverage heading tags (#) to highlight them.

Reply:
Only the name of the project has to be added at the top?